controller/events/dispatcher_api_session.go

```diff
@@ -150,3 +157,24 @@ func (self *apiSessionEventAdapter) IsWrapping(value event.ApiSessionEventHandle
 	}
 	return false
 }
+
+type apiSessionEventOldNsAdapter struct {
+	namespace string
+	wrapped   event.ApiSessionEventHandler
+}
+
+func (self *apiSessionEventOldNsAdapter) AcceptApiSessionEvent(evt *event.ApiSessionEvent) {
+	nsEvent := *evt
+	nsEvent.Namespace = self.namespace
+	self.wrapped.AcceptApiSessionEvent(&nsEvent)
+}
+
+func (self *apiSessionEventOldNsAdapter) IsWrapping(value event.ApiSessionEventHandler) bool {
+	if self.wrapped == value {
+		return true
+	}
+	if w, ok := self.wrapped.(event.ApiSessionEventHandlerWrapper); ok {
+		return w.IsWrapping(value)
+	}
+	return false
+}
```

Comment:
Is this associated with some other issue? The only issue I see mentioned is to add doc, but this also seems to do more?

Did writing the doc reveal the gap between legacy API Sessions vs HA API Sessions?

Reply:
No, this is here to allow the namespace to match the configuration field. So if the config file has
` - edge.apiSessions`, then that'll be the NS in the event. If the config file has `- apiSession`, then that will the NS in the event.